agent/xds/listeners.go

```diff
@@ -1224,54 +1224,56 @@ func makeHTTPFilter(opts listenerFilterOpts) (*envoylistener.Filter, error) {
 	}
 
 	if opts.protocol == "http2" || opts.protocol == "grpc" {
-		cfg.Http2ProtocolOptions = &envoycore.Http2ProtocolOptions{}
+		// TODO(rb): this is deprecated
```

Comment:
Reminder

Reply:
I should go back and clean comment up. It's [freshly deprecated](https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.17.0#deprecated) in `1.17.0`. I originally created this comment because I think it was deprecated in the `go-control-plane` library that I briefly upgraded before reverting due to the protobuf madness.